lib/client-assets.php

```diff
@@ -1189,3 +1189,51 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 	 */
 	do_action( 'enqueue_block_editor_assets' );
 }
+
+/**
+ * Ensure the editor module is loaded before third party plugins.
+ *
+ * Remove this in Gutenberg 3.1
```

Comment:
Should we remove this lines now?

Reply:
Can do, but isn't that technically a task for a separate "Remove Deprecations" PR?

Reply:
According to the github diff, I think this lines:
```
 * Ensure the editor module is loaded before third party plugins.
 *
 * Remove this in Gutenberg 3.1
 */
function polyfill_blocks_module_in_scripts() {
	if ( ! is_gutenberg_page() ) {
		return;
	}
 
	wp_enqueue_script( 'wp-editor' );
}
 
add_action( 'enqueue_block_editor_assets', 'polyfill_blocks_module_in_scripts', 9 );
add_action( 'enqueue_block_assets', 'polyfill_blocks_module_in_scripts', 9 );
...
```

 Are being added in this PR, and they don't look something related to the colors, it was probably a problem during rebase/merge.

Reply:
I removed it myself.

Reply:
Was about to do it. Thank you!

Reply:
Thank you for the changes @gziolo, I was not aware we could add commits in other person forks. That is nice to know 👍

Reply:
Yes, it's ver useful when there is minor update blocking PR. 😃